peachjam/js/components/DocumentProblemModal.vue

```diff
@@ -25,12 +26,45 @@
             :value="url"
           >
           <div class="form-group mb-2">
+            <label for="message">
+              {{ $t('Problem category') }}
+            </label>
+            <select
+              id="message"
+              v-model="selectedOption"
+              class="form-control"
+              name="problem_category"
+              :options="options"
+              required
+            >
+              <option
+                v-for="option in options"
+                :key="option.text"
+                :value="option.value"
+              >
+                {{ option.text }}
+              </option>
+            </select>
+          </div>
+          <div v-if="selectedOption === 'Other'">
```

Comment:
This is overly complicated. The user already has a box they can type in a message -- this field isn't necessary and can be removed.